### PR TITLE
Make sample config json valid

### PR DESCRIPTION
**Pull request by sycdan**
Last night I happened to search for "cli budgeting tool", fully expecting Google to laugh in my face because why would such a thing exist, at which point I would have proceeded to write this exact tool only in Python. However! This was the first hit and I loved it instantly.

I have just started getting it set up, but my first order of business was to make a parser for my Amex statements, and I thought I'd share.

Bonus documentation changes:
- configuration:
  - Add closing square bracket to autoCategorization array in sample config
  - Remove a couple of trailing commas for consistency
  - Use tabs throughout for consistency
  - Remove trailing whitespace
  - Remove extra tab level in `amount` description
- usage:
  - remove unused `--input` argument name

### CODE REVIEW

**Comment by joshcanhelp** [comment]
@sycdan - I'm so sorry for the delay here! Thank you so much for your contribution! I'm right in the middle of getting another project out but I'll come back and review in the next couple of days. Thank you for making my day with a PR!

File: docs/configuration.md [hunk 1]
```diff
@@ -4,80 +4,81 @@ This tool can be configured using a `.budget-cli.json` file in the root of this
 
 ```json
 {
-  "outputFile": {
```
Comment (joshcanhelp) [inline docs/configuration.md]: It looks like the spacing here went from spaces to tabs, making it hard to see any changes that might have been made.

**Comment by joshcanhelp** [comment]
If you could check the minor comments here and run the listing/formatting locally, I'm fine with merging:

https://github.com/joshcanhelp/budget-cli/blob/main/CONTRIBUTING.md#i-want-to-make-code-changes

Thanks again @sycdan!

**Comment by sycdan** [comment]
Awesome, will try to publish the cleanup soon!

**Comment by joshcanhelp** [comment]
@sycdan - Hey there ... just checking in on this. If you're not able to look at the spacing in the docs, I can give it a look. Thanks!

**Comment by sycdan** [comment]
Sorry, I thought I had fixed up what things you pointed out. What would you like me to do? Change tabs to spaces?

**Review by joshcanhelp — APPROVED** [review]

**Comment by joshcanhelp** [comment]
@sycdan - Thanks again for this!

**Pull request merged**